react/week3/src/App.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import Header from './components/Header';
+import AddRandom from './components/AddRandom';
+import TodoForm from './components/TodoForm';
+import Todo from './components/Todo';
+import './App.css';
+
+function App() {
+	const [ todos, setTodos ] = useState([
+		{
+			text: 'Get out of bed',
+			deadline: '16-02-2020',
+			isCompleted: false
+		},
+		{
+			text: 'Brush teeth',
+			deadline: '16-03-2020',
+			isCompleted: false
+		},
+		{
+			text: 'Eat breakfast',
+			deadline: '20-03-2020',
+			isCompleted: true
+		}
+	]);
+
+	const updateTodo = (index, text, deadline) => {
+		const updatedTodos = todos.map((element, currentIndex) => {
+			if (currentIndex == index) {
+				return {
+					...element,
+					text: text,
+					deadline: deadline
+				};
+			} else return element;
```

Comment:
There is no need to `else` block here
```suggestion
			if (currentIndex === index) {
				return {
					...element,
					text: text,
					deadline: deadline
				};
			}
			
			return element;
```